### ISSUE TITLE
Slash command for output from tasks `/task`

### ISSUE THREAD

**Issue by oliver-kriska** [zed-industries/zed#17001]
### Check for existing issues

- [X] Completed

### Describe the feature

Same like we have `/tab`  or `/terminal` we should have also `/task` so we use output from task in Assistant. I use tasks for Elixir development so I have tasks for tests or `mix dialyzer`  (in my case custom alias for formatter, credo and dialyzer running as one task).  Probably it will need two attributes for selecting which task output to use and maybe lines same like with terminal.

**Comment by osiewicz** [comment]
You can already use `/terminal` to grab task output (though - admittedly - I've just learned that myself). We should definitely document that.

**Comment by notpeter** [comment]
We have docs! https://zed.dev/docs/assistant/commands#terminal

**Comment by osiewicz** [comment]
Yeah, but they don't mention that you can use `/terminal` for tasks

**Comment by oliver-kriska** [comment]
thanks, it's not mention anywhere so linking that docs was really useless.

Also I tried it. It works only with "last active" terminal. So when I use task and use also actual terminal for something else (REPL, zsh,...) it will not use that task's terminal but lastly used. So it's basically working by accident let's say. 

So maybe it would be nice solution to be able select which terminal tab I want to use. That will be working for Tasks and also for people who use multiple terminals at all.

**Comment by notpeter** [comment]
We're not actively prioritizing the development of slash command for Text Threads.
With the agent panel this is less needed because the agent can directly invoke terminal commands and see their output as normal tool calls.

Thanks for reporting.
